### PR TITLE
Add the ChronoshiftPower

### PR DESCRIPTION
**Pull request by abcdefg30**
Thanks to @tomsons26 for the basic work on this.

### CODE REVIEW

File: rules/allied-structures.yaml [hunk 1]
```diff
@@ -760,6 +760,17 @@ gacsph:
...
+	ChronoshiftPower@chronoshift:
+		OrderName: Chronoshift
+		Icon: chrono
+		ChargeTime: 120
+		Description: Chronoshift
+		LongDesc: Teleports a group of units across\nthe map.
+		# Those sounds don't work: ChunkSize is 0
+		#SelectTargetSound: schrlo1.wav, schrlo2a.wav schrlo2b.wav, schrlo2c.wav, schrlo3.wav
+		EndChargeSound: schropen.wav
+		KillCargo: yes
```
Comment (GraionDilach) [inline rules/allied-structures.yaml]: Wrong!

**Comment by GraionDilach** [comment]
Notsure if default values trigger it, but RA2 Chronosphere doesn't bring back the units after a time and it only kills directly targeted infantry (excluding CLeg), not passengers.

**Comment by abcdefg30** [comment]
> Notsure if default values trigger it, but RA2 Chronosphere doesn't bring back the units after a time 

I know, see https://github.com/OpenRA/ra2/pull/52/files#diff-ffa2589dfb42cb2d2ecffa68264dd139R409.

File: rules/allied-structures.yaml (hunk 1 above)
Comment (abcdefg30) [inline rules/allied-structures.yaml]: Removed it.

**Comment by abcdefg30** [comment]
Just had an idea. We can make use of the `ExplodeInstead` flag on `Chronoshiftable` (which just kills stuff and doesn't require an explosion) to add the correct behaviour for infantry. They don't show a death animation yet though.

**Comment by GraionDilach** [comment]
That sounds good to me as a start.

**Comment by GraionDilach** [comment]
Oh, I see they have been added already. :+1: then but wait with this for someone else.

File: rules/allied-structures.yaml [hunk 2]
```diff
@@ -760,6 +760,16 @@ gacsph:
...
 	RequiresPower:
 	CanPowerDown:
+	ChronoshiftPower@chronoshift:
+		OrderName: Chronoshift
+		Icon: chrono
+		ChargeTime: 120
+		Description: Chronoshift
+		LongDesc: Teleports a group of units across\nthe map.
+		# Those sounds don't work: ChunkSize is 0
+		#SelectTargetSound: schrlo1.wav, schrlo2a.wav schrlo2b.wav, schrlo2c.wav, schrlo3.wav
```
Comment (Mailaender) [inline rules/allied-structures.yaml]: Filed https://github.com/OpenRA/OpenRA/issues/10540.

**Comment by Mailaender** [comment]
I submitted https://github.com/abcdefg30/ra2/pull/1 to fix the crash I introduced recently.

> Exception of type `System.InvalidOperationException`: Unit `overlay` does not have a sequence named `target-valid-temperate`

**Comment by abcdefg30** [comment]
Updated.

File: rules/allied-structures.yaml (hunk 1 above)
Comment (abcdefg30) [inline rules/allied-structures.yaml]: Actually, the default here is already `true`. Fixed that in #70.

**Pull request merged**